vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/TimelineItemFactory.kt

```diff
@@ -113,14 +113,15 @@ class TimelineItemFactory @Inject constructor(
                     EventType.CALL_NEGOTIATE,
                     EventType.REACTION,
                     in EventType.POLL_RESPONSE,
-                    in EventType.POLL_END           -> noticeItemFactory.create(params)
+                    in EventType.POLL_END             -> noticeItemFactory.create(params)
                     // Calls
                     EventType.CALL_INVITE,
                     EventType.CALL_HANGUP,
                     EventType.CALL_REJECT,
-                    EventType.CALL_ANSWER           -> callItemFactory.create(params)
+                    EventType.CALL_ANSWER             -> callItemFactory.create(params)
+                    in EventType.BEACON_LOCATION_DATA -> noticeItemFactory.create(params)
```

Comment:
Should be grouped with other NoticeItem

Reply:
Fixed see [2aeee79](https://github.com/vector-im/element-android/pull/5989/commits/2aeee79c63bdc78421254e8db630ccfbb1fcb873)